app/src/main/java/openfoodfacts/github/scrachx/openfood/views/ProductActivity.java

```diff
@@ -97,7 +97,9 @@ private void setupViewPager(ViewPager viewPager) {
         String[] menuTitles = getResources().getStringArray(R.array.nav_drawer_items_product);
 
         ProductFragmentPagerAdapter adapterResult = new ProductFragmentPagerAdapter(getSupportFragmentManager());
-        adapterResult.addFragment(new SummaryProductFragment(), menuTitles[0]);
+        SummaryProductFragment summaryProductFragment = new SummaryProductFragment();
+
+        adapterResult.addFragment(summaryProductFragment, menuTitles[0]);
         adapterResult.addFragment(new IngredientsProductFragment(), menuTitles[1]);
```

Comment:
Not in the order of display : Front, Ingredient, Nutrition here (which is the right thing), 
Actually displayed: Front, Nutrition, Ingredients